Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.1.inc

```diff
@@ -60,7 +60,7 @@ class EmptyHandler
  * @api is ok here
  * @deprecated can be used in this context
  * @see is ok here
- * @author is actually ok
+ * @author is not ment to be used
```

Comment:
```suggestion
 * @author is not meant to be used
```

Reply:
Thanks for spotting this. I copied the text from two lines below and didn't spot this mistake in the original. I've fix this and the original now.